src/MassTransit/Automatonymous/MassTransitStateMachine.cs

```diff
@@ -219,10 +219,7 @@ protected override void Event<T>(Expression<Func<Event<T>>> propertyExpression)
                     .CancelRequestTimeout(request)
                     .ClearRequest(request),
                 When(request.Faulted)
-                    .CancelRequestTimeout(request)

```

Comment:
I'm going to adjust this slightly.

Reply:
You're absolutely welcome.

Reply:
Actually, the diff confused me, looks fine in the code. Thanks again!

Reply:
I was wondering, because it did the same to me 😆.